ui/components/ConfirmationModal.js

```diff
@@ -14,7 +14,8 @@ import { getK8sConfigIdsFromK8sConfig } from "../utils/multi-ctx";
 import { bindActionCreators } from "redux";
 import { useEffect, useState } from "react";
 import DoneAllIcon from '@material-ui/icons/DoneAll';
-import UndeployIcon from "../public/static/img/UndeployIcon";
+// import UndeployIcon from "../public/static/img/UndeployIcon";
+import RemoveDoneIcon from '@mui/icons-material/RemoveDone';
```

Comment:
changed this to mui because we are using central styling for mui icons from app.js and rest are also mui icons so i didn't changed that use case and to consistent i bring here mui icon

Reply:
The issue is that people don't ensure that MUI icon scale in Safari. Let's not change it if its not causing a problem.

Reply:
Somewhere it is causing issues the color reference is not effecting, where as others are working fine.

Reply:
<img width="1110" alt="image" src="https://github.com/meshery/meshery/assets/51183483/03ee9ebc-67bd-4e9b-ae03-fe6cb707d116">
Scaling for me in safari above picture is zoomed 90%

Reply:
SVG scaling system is added to the main theme https://github.com/meshery/meshery/pull/8229, in Mehsery UI we should not get any scaling issues with MUI icons. // @leecalcote